homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -147,31 +172,161 @@ class XiaomiMiioSensorDescription(SensorEntityDescription):
         icon="mdi:fast-forward",
         state_class=STATE_CLASS_MEASUREMENT,
     ),
+    ATTR_MOTOR2_SPEED: XiaomiMiioSensorDescription(
+        key=ATTR_MOTOR2_SPEED,
+        name="Second Motor Speed",
+        native_unit_of_measurement="rpm",
+        icon="mdi:fast-forward",
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
     ATTR_ILLUMINANCE: XiaomiMiioSensorDescription(
         key=ATTR_ILLUMINANCE,
         name="Illuminance",
         native_unit_of_measurement=UNIT_LUMEN,
         device_class=DEVICE_CLASS_ILLUMINANCE,
         state_class=STATE_CLASS_MEASUREMENT,
     ),
+    ATTR_ILLUMINANCE_LUX: XiaomiMiioSensorDescription(
+        key=ATTR_ILLUMINANCE,
+        name="Illuminance",
+        native_unit_of_measurement=LIGHT_LUX,
+        device_class=DEVICE_CLASS_ILLUMINANCE,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
     ATTR_AIR_QUALITY: XiaomiMiioSensorDescription(
         key=ATTR_AIR_QUALITY,
         native_unit_of_measurement="AQI",
         icon="mdi:cloud",
         state_class=STATE_CLASS_MEASUREMENT,
     ),
+    ATTR_PM25: XiaomiMiioSensorDescription(
+        key=ATTR_AQI,
+        name="PM2.5",
+        native_unit_of_measurement=CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+        icon="mdi:blur",
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    ATTR_FILTER_LIFE_REMAINING: XiaomiMiioSensorDescription(
+        key=ATTR_FILTER_LIFE_REMAINING,
+        name="Filter Life Remaining",
+        native_unit_of_measurement=PERCENTAGE,
+        icon="mdi:air-filter",
+        state_class=STATE_CLASS_MEASUREMENT,
+        attributes=(
+            "filter_rfid_tag",
+            "filter_type",
```

Comment:
Do we need these attributes? Can they change during the entity lifetime and what values can they take?

Reply:
The `filter_rid_tag` looks like a MAC address (f.e. `81:66:6a:da:85:a:4`). If you change the filter once a year the value changes. The `filter_type` is probably static (if you buy always the same filter type). 3 different values ("regular", "anti-bacterial", "anti-formaldehyde") are supported/known: https://github.com/rytilahti/python-miio/blob/master/miio/airfilter_util.py#L13

Reply:
I think `filter_type` could be useful for the users. `filter_rid_tag` can be removed. What do you think?

Reply:
Ok.